ci/docker/get_data_volumes.py

```diff
@@ -4,7 +4,8 @@
 import subprocess
 import shlex
 
-METPLUS_VERSION = 'develop'
+#METPLUS_VERSION = 'develop'
```

Comment:
We need to be sure that we update this back to develop once those are all available or else new use cases will fail.